### DISCUSSION TITLE
Integrations missing in v0.66

### DISCUSSION THREAD

**Discussion by MuratDoganer** [outline/outline#4084] [Self Hosting] [answered]
Hey All

I just finished installing Outline, and noticed that all of the integrations are missing. I was hoping to set up Slack but I get a "page not found" error when searching for `settings/integrations/slack`

Is it only available on the cloud platform?

**Comment by tommoor** [comment]
It's available, my guess is that your account is not an admin user for some reason.

**Reply by MuratDoganer** [reply]
That’s very odd, in the settings under users it shows me as admin (I’m the only user as well for now)

**Reply by tommoor** [reply]
Oh, you have to have configured the various `SLACK_` environment variables for it to show up

**Reply by MuratDoganer** [reply]
Hmm i thought I had (I used the sample env linked in the documentation) 

was there any more that might not be in the sample env?

**Comment by MuratDoganer** [comment, marked as answer]
Does anyone have any idea what I need to do to get Integrations?

Still same issue, integrations are missing apart from 2

![Screenshot 2022-09-12 at 16 04 32](https://user-images.githubusercontent.com/67741877/189689140-6a15c574-32f4-4dab-8acc-b79db1e969d0.png)

![Screenshot 2022-09-12 at 16 04 50](https://user-images.githubusercontent.com/67741877/189689178-df062fc5-b901-4319-9b9b-cb7dd9c2dccf.png)

**Reply by MuratDoganer** [reply]
I've even reinstalled Outline to see if that was the issue, second time round and still no available Integrations

**Reply by Limezy** [reply]
It's normal if you see only these two integrations on the sidebar. Only these two have dedicated settings. 
Have you... just tried to use an integration using the editor in a new doc ?

Edit : sorry, I've read too fast. Indeed settings should probably show up for slack if you setup env accordingly. 
Have you tried to set the logs to debug ? Nothing showing in the console ?

**Reply by MuratDoganer** [reply]
Nothing in logs 😔 Slack is definitely set up in Env (using the sample env) but I’m wondering if there were any additional ones added that I may not be aware of

**Reply by tommoor** [reply]
It's literally enabled by the existence of SLACK_CLIENT_ID, that's all it takes

**Reply by MuratDoganer** [reply]
Yup found the problem, I had `CLlENT_KEY=`

The `I` in client was a lower case `l`, not an uppercase I. 

oops.
